### ISSUE TITLE
안녕하세요. 책과 Github의 코드가 다른 곳을 찾았습니다.

### ISSUE THREAD

**Issue by arer90** [wikibook/tf2#6]
안녕하세요, 개발자님 덕분에 deep learning에 대해서 쉽게 적근하고 있는 대학원생(석사3기) 신생 개발자입니다. 개발자님의 친절한 소개와 개념에 대한 간단한 예시들이 많은 도움을 얻었습니다. 본론으로 들어와서 책에서 5장 '와인 데이터세트 분류 모델 생성'에서 116 pg에 

model.compile(optimizer=tf.keras.optimizers.Adam(lr=0.07)),
loss='sparse_categorical_crossentropy', metrics=['accuracy']) 

로 적혀져 있으나, Github에서는 

model.compile(optimizer=tf.keras.optimizers.Adam(lr=0.07), loss='categorical_crossentropy', metrics=['accuracy'])

으로 'sparse'라는 단어가 없다는 것을 확인되었습니다. 현재 개인적으로 사용하는 것은 anaconda의 spyder 4이지만 google colab에서도 유사한 문제가 발생하여 이렇게 글을 올립니다. 사소한 제 의견이지만 너그럽게 봐주시고, 앞으로도 잘 부탁드립니다.

**Comment by greentec** [comment]
안녕하세요. 책이 도움이 되고 있다니 기쁩니다.
말씀주신 사항은 https://github.com/wikibook/tf2/issues/5 에서 발견되어 Github에서 수정된 사항으로 2쇄가 나오면 책 본문에도 수정 반영될 예정입니다. 피드백주셔서 감사합니다.
